### ISSUE TITLE
Underground biomes causing crashes

### ISSUE THREAD

**Issue by shucktech** [WesCook/Homestead#65]
Since the update, my server has become rather unstable. The map doesn't have that much going on in it, but I keep getting the following error every now and then: https://i.imgur.com/qlRpgOx.png

It's causing the server to shut down. Any thoughts? Also, thanks for the responses on the last issue I posted.

**Comment by WesCook** [comment]
Confirmed.  Thanks for the report.

Oof, that's a bad one.

**Comment by WesCook** [comment]
Perhaps not a bug after all.  While I saw this same error on an initial run of the server, it looks like it's being caused by the EULA check not succeeding.

Could you verify if you've set eula=true under eula.txt?

Unfortunately the error message isn't very descriptive.  It implies that a mod error occurred instead, which may lead to confusion.  Not much to be done about that other than including the eula file by default, which may be frowned upon.

**Comment by shucktech** [comment]
I can confirm the elua is there, and enabled: https://i.imgur.com/BS62TEs.png

The crash happens every now and then while users are playing on the server. I have not seen it take place while launching the server. 

PS let me know where to look to get you more info.

**Comment by shucktech** [comment]
Server crashed again, pulled a bit more information, let me know if that's more of what we need to figure this out or not. Good thing I got my auto restart script working. 

Crash Log: https://pastebin.com/Ugrq1UAb
Update: I'm thinking it might have something to do with saltlicks? https://i.imgur.com/3susXVr.png

**Comment by WesCook** [comment]
Ah, thank you.  That is very useful information.

Do you have a trough directly beside a salt lick?  It looks like there's an Animania crash there.

https://github.com/capnkirok/animania/issues/436

Though reading the changelog for 1.4.2, it suggests the fix is 1.12 only:

>[1.12 only] Salt Licks no longer crash server and ruin world when placed next to troughs (credit: Nerezza)

I can test this out shortly to see if the fix still works in 1.10.  But if you'd like to confirm the fix right away, you can upgrade your version of Animania used in the pack.

**Comment by shucktech** [comment]
I can confirm, we had some next to the troughs. I removed all the ones (that I know of.) However haven't tested the pack much since removing them to confirm if that resolved the issue or not. 

Also, let me know what you find and I'll update items as needed if you come to a final resolution. Luckily, my community is pretty small, so I've just instructed everyone not to make any salt licks for the time being.
